foursight_core/fs_connection.py

```diff
@@ -40,6 +44,12 @@ def __init__(self, fs_environ, fs_environ_info, test=False, use_es=True, host=No
         self.ff_env = fs_environ_info['ff_env']
         self.ff_es = fs_environ_info['es']
         self.ff_bucket = fs_environ_info['bucket']
+        if 'redis' in fs_environ_info:
+            self.redis = RedisBase(create_redis_client(url=fs_environ_info['redis']))
```

Comment:
Why a self.redis here and a self._redis in related classes? And are these redundant objects or serving different purposes? It's confusing to have different kinds of things having essentially the same name.

Reply:
They are separate uses as the React version doesn't actually call into here directly, so a handler needs to be resolved under both schemes to support Redis session in the legacy and React versions.